src/Microsoft.Azure.WebJobs.Logging/ILogTableProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    /// <summary>
+    /// Logs are stored across multiple tables to aide in purging old logs.  
```

Comment:
Please update the xml comments on these public types to be what you'd expect to see as a 3rd party outsider. E.g., for this interface, I'd expect something like "Defines an interface for managing log tables.".